### PR TITLE
Unlimited assets changes

### PR DESCRIPTION
**Pull request by tolikzinovyev**
Unlimited assets changes.

Closes https://github.com/algorand/go-algorand-internal/issues/1862.

### CODE REVIEW

File: client/v2/indexer/lookupAccountByID.go [hunk 1]
```diff
@@ -29,6 +34,14 @@ type LookupAccountByID struct {
 	p LookupAccountByIDParams
 }
 
+// Exclude exclude additional items such as asset holdings, application local data
+// stored for this account, asset parameters created by this account, and
+// application parameters created by this account.
+func (s *LookupAccountByID) Exclude(Exclude []string) *LookupAccountByID {
```
Comment (nicholasguoalgorand) [inline client/v2/indexer/lookupAccountByID.go:40]: why is this a list of strings?

Comment (tolikzinovyev) [reply client/v2/indexer/lookupAccountByID.go:40]: This code is generated. I didn't have a lot of choice...

Comment (winder) [reply client/v2/indexer/lookupAccountByID.go:40]: You can provide multiple Exclude values, i.e. `created-assets,created-apps`

Comment (tolikzinovyev) [reply client/v2/indexer/lookupAccountByID.go:40]: Do you mean 4 arguments for `Exclude()`?

File: test/responses_unit_test.go [hunk 2]
```diff
@@ -112,7 +125,7 @@ func weMakeAnyCallTo(client /* algod/indexer */, endpoint string) (err error) {
 				LastRound:        uint64(sParams.FirstRoundValid),
 				MinFee:           sParams.MinFee,
 			}
-		case "GetAccountInformation":
+		case "AccountInformation":
```
Comment (winder) [inline test/responses_unit_test.go:128]: Why does this change?

**Review by winder — COMMENTED** [review]
These changes look good, I just had one question.

File: test/responses_unit_test.go (hunk 2 above)
Comment (tolikzinovyev) [reply test/responses_unit_test.go:128]: For consistency I think.

**Comment by tolikzinovyev** [comment]
The CI passes, please approve.

**Pull request merged**